code/game/mecha/mecha.dm

```diff
@@ -669,13 +669,13 @@
 			AI.remote_control = null
 			icon_state = initial(icon_state)+"-open"
 			to_chat(AI, "You have been downloaded to a mobile storage device. Wireless connection offline.")
-			to_chat(user, "<span class='boldnotice'>Transfer successful</span>: [AI.name] ([rand(1000,9999)].exe) removed from [name] and stored within local memory.")

```

Comment:
this is a proper use of name

Reply:
This shows not ok here but when editing it's ok?